src/containerapp/azext_containerapp/_params.py

```diff
@@ -344,3 +344,6 @@ def load_arguments(self, _):
         c.argument('environment_name', options_list=['--environment'], help="The environment name.")
         c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None)
         c.argument('configuration', nargs="*", help="Java component configuration. Configuration must be in format \"<propertyName>=<value> <propertyName>=<value> ...\".")
+
+    with self.argument_context('containerapp env', arg_group='Peer Traffic Configuration') as c:
+        c.argument('p2p_encryption_enabled', arg_type=get_three_state_flag(), options_list=['--enable-peer-to-peer-encryption'], help='Boolean indicating whether the peer-to-peer traffic encryption is enabled for the environment.')
```

Comment:
The param's name is pretty long, recommend giving a shorter name too.

Reply:
This parameter name is already signed off by PM. We have to keep it and exclude the warning from the linter.

Reply:
I mean to can provide an additional short name.